### DISCUSSION TITLE
Pink Color Issue in Full Screen (for macOS Sonoma version 14.5)

### DISCUSSION THREAD

**Discussion by apartmandairesi** [nicotine-plus/nicotine-plus#3065] [Q&A] [answered]
Hello.

I am using **macOS Sonoma version 14.5**. I have tried to install Nicotine+-3.3.4. However when I've opened app, I've saw an error which Is **pink color** in full screen. Do you have any idea about this issue? Thank you.

**Comment by slook** [comment]
No idea. Please post a screen shot.

**Comment by apartmandairesi** [comment]
here is screen recorder.


https://github.com/user-attachments/assets/aa4fe51f-11d2-4e7d-b152-d8df1b0f1420

**Comment by slook** [comment]
It looks like some kind of transparency placeholder generated by Gtk/Gdk but I have no clue what would cause that for sure.

Is the [latest development build](https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md#macos) also affected by the same issue?

Also, please clarify more information about your hardware, what type of Mac is it?

**Comment by apartmandairesi** [comment]
Yes, I have tried to install with 3.3.5. + 3.3.4  versions with seperately and same problem is still continue.

My hardware is MacBook Pro, 13-inch, M1, 2020, 16 GB Ram, 1 TB HDD, using Sonoma 14.5.

**Comment by mathiascode** [comment]
Does https://gaphor.org/ (another GTK 4 app) have the same issue?

**Comment by apartmandairesi** [comment]
I just installed this one and it works very well ) :

**Comment by slook** [comment]
Is it possible for you to start `nicotine` from the Terminal, and see if there are any error messages on the console?

**Comment by apartmandairesi** [comment]
Sure but I have no idea that how can I start Nicotine from the Terminal? What should I write?

**Reply by slook** [reply]
I'm not sure, it might be something like:

`open -a Nicotine+.app`

**Comment by apartmandairesi** [comment]
still continue

https://github.com/user-attachments/assets/52270183-41f7-4edc-a26e-6649796a023e

**Reply by slook** [reply]
The command I suggested doesn't have the needed effect, nothing is printed into the Terminal console window.

It is necessary to run the `nicotine` script directly, but I don't know how this is accomplished on your platform.

**Comment by apartmandairesi** [comment]
Ok, thank you so much.

**Comment by mathiascode** [comment]
Try the following:
- Right-click the Nicotine+ icon and select "Show package contents"
- Navigate to the `Contents/MacOS` folder
- Right-click the `Nicotine+` executable, and select the option to open in a terminal

Then paste the output of the terminal here.

Also, does anything happen if you try to drag the Nicotine+ window from the top left corner?

**Reply by mathiascode** [reply]
Another thing, have you noticed this issue anywhere else in the OS? I wonder if it's some issue related to your GPU, since nobody else has reported this issue so far.

**Reply by slook** [reply]
According to [Apple MacBook Pro (13-inch, M1, 2020) Technical Specifications](https://support.apple.com/en-us/111893) this machine has an integrated GPU and was originally supplied with Monterey, so there is a possibility the vendor has made an oversight in backward compatibility with Gtk applications following an OS update.

It would be useful to do a test with another Gtk 4 app, or otherwise edit the app launcher to run Nicotine+ with Gtk 3.

**Reply by apartmandairesi** [reply]
> Try the following:
> 
> * Right-click the Nicotine+ icon and select "Show package contents"
> * Navigate to the `Contents/MacOS` folder
> * Right-click the `Nicotine+` executable, and select the option to open in a terminal
> 
> Then paste the output of the terminal here.
> 
> Also, does anything happen if you try to drag the Nicotine+ window from the top left corner?

I am adding a screenshot about what's happening after the following:

<img width="1145" alt="Screenshot at Jul 26 20-18-17" src="https://github.com/user-attachments/assets/7eb6c80c-54aa-4599-b22d-34121fbc0442">


Also, does anything happen if you try to drag the Nicotine+ window from the top left corner?
< When I've tried this I saw Nicotine+ app and pink color issue is gone however I've back to same full screen pink issue after 2-3 seconds.

**Comment by slook** [comment]
@apartmandairesi Please try the latest build of Nicotine+ 3.3.5rc2 to verify if the problem still exists or not?...
https://github.com/nicotine-plus/nicotine-plus/blob/master/doc/TESTING.md#macos

**Comment by apartmandairesi** [comment]
Hello. I am so sorry but I would like to confirm that problem is still continue with Nicotine+ 3.3.5rc2.

**Reply by slook** [reply]
Okay, it doesn't seem as if all "Sonoma" users are affected by this problem, otherwise we would have had more reports.

It is either all "MacBook Pro M1 2020 Sonoma" users, or just you... We have no way to know how widespread the problem is for sure because the application doesn't have any telemetry.

The bug cannot be narrowed down any further until we know this.

**Reply by apartmandairesi** [reply]
Thank you so much for your kindly reply.

**Reply by slook** [reply]
@apartmandairesi Do you use an app called "Better Display" (as discussed in [issue #3066](https://github.com/nicotine-plus/nicotine-plus/issues/3066#issuecomment-2393871171)) for non-native resolution scaling?

**Reply by apartmandairesi** [reply]
Wow, yes, it works with very well. Thank you.

**Reply by slook** [reply]
Are you saying that you do have this app installed? If so, did setting it differently or uninstalling it help you to avoid the bug in some way?

**Reply by apartmandairesi** [reply, marked as answer]
I've installed the app which is called Better Display and when I've off the button of High Resolution (HiDPL), issue is gone. Everything works perfect.

**Reply by apartmandairesi** [reply]
+ when I've ON the button of High Resolution (HiDPL) also Nicotine works with very well. Just to know.
